### ISSUE TITLE
Emission/Resource mismatch for CO2 to soil or biomass stock

### ISSUE THREAD

**Issue by jedesroches** [ecoinvent/lcia#5]
Substances `259cf8d6-6ea8-4ccf-84b7-23c930a5b2b3`, `8ae4d8bb-3e4b-4825-8325-94696d7a64fd`, `60d424f7-d5a9-4549-9540-da06684bc3bb`, `375bc95e-6596-4aa1-9716-80ff51b9da77` are all used as outgoing elementaryExchange (with `<outputGroup>4</>`), but are mapped to compartements/sub- that are resources rather than emissions: `Resources from air,Renewable material resources from air`

**Comment by thomassonderegger** [comment]
Indeed, this is an inconsistency. However, the way it is treated in the mappings is ok: this is CO2 bound in soils, so rather a negative emission than a resource use. This is modeled a bit differently in different data systems (such as EF and ecoinvent). The important thing is that the flow (which has a positive sign in datasets) gets a negative characterization factor.

**Comment by cmutel** [comment]
I guess part of the problem here is our sign convention for resources, where a positive number is an extraction of a resource.

What would the ideal version be? I think:
* Either CO2 in the air (resource) is extracted and carbon in the soil is increased, or
* A negative CO2 emission is tied to an increase in soil organic carbon.

In either case we need to get the mass balance correct, and make sure the LCIA methods don't double count on both flows.

**Comment by thomassonderegger** [comment]
Both options outlined by @cmutel are considered in land use modelling if I don't get it wrong: CO2 to soil needs to come from somewhere (Carbon dioxide, in air) and CO2 from soil also needs to come from somewhere (Carbon, organic, in soil or biomass stock). So source and destination are clearly defined, it is less clear whether and how to categorize this in a resource-emission dichotomy. A resource extraction is not really the same as a negative emission, and carbon in soil is not really a resource. The choice was to characterize what is labelled "ToEnvironment". This is consistent and the mapping to EF has been done according to this choice. Thus, while you might create you own mapping, I would not accept a PR for now. We might change this in the future (or not :-)).
